test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/TimestampProviderTests.cs

```diff
@@ -139,10 +137,10 @@ public async Task GetTimestamp_WhenRequestNull_ThrowsAsync()
             var authorCertName = "author@nuget.func.test";
             var content = new SignatureContent(SigningSpecifications.V1, Common.HashAlgorithmName.SHA256, "Test data to be signed and timestamped");
 
-            Action<X509V3CertificateGenerator> modifyGenerator = delegate (X509V3CertificateGenerator gen)
+            Action<TestCertificateGenerator> modifyGenerator = delegate (TestCertificateGenerator gen)
             {
-                gen.SetNotBefore(DateTime.MinValue);
-                gen.SetNotBefore(DateTime.UtcNow.Subtract(TimeSpan.FromDays(1))); // cert has expired
+                gen.NotBefore = DateTime.MinValue;
```

Comment:
Is this line necessary?  It looks like the value is being overwritten by the next statement.